python/tchannel/thrift/scheme.py

```diff
@@ -28,7 +28,7 @@
 from .. import scheme
 
 
-class ThriftArgScheme(scheme.ArgScheme):
+class ThriftArgScheme(scheme.RawArgScheme):
```

Comment:
You can image a custom argscheme only implements its own serialization/de-serialization  for header for example, then it doesn't need to care about the body part.